Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -733,7 +733,7 @@ extension CarPlayManager: CPMapTemplateDelegate {
         if let passiveLocationProvider = navigationMapView.mapView.location.locationProvider as? PassiveLocationProvider {
             passiveLocationProvider.locationManager.resumeTripSession()
         }
-        delegate?.carPlayManagerDidEndNavigation(self)
```

Comment:
No longer calling this method would be a backwards-incompatible change. We should continue to call it but informally encourage developers to use the other version.